Cargo.toml

```diff
@@ -77,5 +78,10 @@ path = "examples/06_material/main.rs"
 name = "sphere_separate"
 path = "examples/07_separate_sphere/main.rs"
 
+[[example]]
+name = "gltf"
+path = "examples/08_gltf/main.rs"
+
 [workspace]
-members = ["amethyst_assets", "amethyst_audio", "amethyst_config", "amethyst_core", "amethyst_input", "amethyst_renderer", "amethyst_utils"]
+members = ["amethyst_assets", "amethyst_audio", "amethyst_config", "amethyst_core", "amethyst_gltf", "amethyst_input",
```

Comment:
We should eventually make this a virtual workspace. I think those support kind of implicit members, not sure.

Reply:
No idea tbh :)